nifcloud/nifcloud-cli
# Feature/windows installer

## Summary

- Support Windows MSI Installer.

## Tests

*Must be test on a Windows machine.*

- [x] install pipenv
- [x] `pipenv install -d --skip-lock`
- [x] `pipenv run python setup.py build` 
    - check the `build/exe.win-amd63-*` directory
- [x] `pipenv run python setup.py bdist_msi`
    - check the `dist` directory
- [x] Install nifcloud-cli via MSI installer in `dist` directory.
